specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-10-01/managedClusters.json

```diff
@@ -1772,6 +1772,11 @@
           "type": "string",
           "description": "FQDN for the master pool."
         },
+        "privateFQDN": {
```

Comment:
Please, add an example how to use the property.

Reply:
@sergey-shandar added